### ISSUE TITLE
[Fishing] lvl 50+ Zones with wrong fish loot tables and wrong drop rates

### ISSUE THREAD

**Issue by ghuser44** [SallyWhitemane/Mograine#858]
**Description:**
The waters where you can fish in all lvl 50+ zones are messed up and have the wrong fishes, with the wrong drop rates. As an example: I've been trying to fish **Darkclaw Lobster** (~40% drop rate) and **Large Raw Mightfish** (~5% drop rate) in Azshara, in the specific locations where you're supposed to fish them (a quick search on youtube and wowwiki will show where) without any success, for an hour, now. All I get is stonescale eel (which is supposed to only have around 18% drop rate) and Zesty clam meat.

**How to reproduce:**
Fish in any zone with lvl 50+. Azshara, for example. Try to get **Darkclaw Lobster**, for example.

**How it should work:**
the lvl 50+ zones should have the correct fishes in them.

**Comment by alexspew** [comment]
Hi, this was related to a bug with the loot in Azshara, Darkclaw Lobster was messing up. So I've added it outside as it's own drop, please try again later today (Since the server crashed, it should've been applied). If there's anything, contact me on discord : alexs
Thanks!

**Comment by ghuser44** [comment]
I successfully fished **Darkclaw Lobster** from both cave locations, in Bay of storms, in Azshara.

However, I also fished a lot of low level fish, like *Raw spotted yellowtail* and *Raw mithril head trout*, most often on the same loot table. Also, I couldn't fish any **Large Maw Mightfish** or **Raw Whitescale salmon** in either Azshara or Winterspring. In Winterspring, in the water near the west entry, I could only fish *Raw Nightfin sapper* and *Raw Redgill*, even though I'm 342 in fishing and I'm using Bright Baubles bait.
